lightning/src/routing/router.rs

```diff
@@ -314,6 +314,10 @@ fn compute_fees(amount_msat: u64, channel_fees: RoutingFees) -> Option<u64> {
 
 /// Gets a route from us (payer) to the given target node (payee).
 ///
+/// If the payee provided features in their invoice, they should be provided via payee_features.
+/// Without this, any knowledge we have of the payee's features may be used if they are available
+/// in the network graph, however MPP routing will be disabled and routing is more likely to fail.
```

Comment:
Maybe a bit clearer as:

s/features may be used if they are available in the network graph/features may be used in the network graph if they are available

Or set off "if they are available" as "(if available)".

Reply:
I rewrote it a bunch more.